Examples/Framework/src/Framework/Sequencer.cpp

```diff
@@ -8,17 +8,24 @@
 
 #include "ActsExamples/Framework/Sequencer.hpp"
 
+#include "Acts/Utilities/Helpers.hpp"
 #include "ActsExamples/Framework/ProcessCode.hpp"
 #include "ActsExamples/Framework/WhiteBoard.hpp"
 #include "ActsExamples/Utilities/Paths.hpp"
 
 #include <algorithm>
 #include <atomic>
+#include <cfenv>
 #include <chrono>
 #include <exception>
+#include <limits>
 #include <numeric>
 #include <stdexcept>
 
+#include <math.h>
+#include <stdio.h>
+#include <stdlib.h>
```

Comment:
should we replace these c includes with c++ ones?

Reply:
Yeah I think this is just again my vim auto-including.